--- include/rootfs.mk	2025-02-04 18:34:49.774419185 +0000
+++ include/rootfs.mk	2025-02-04 18:36:56.902485860 +0000
@@ -93,11 +93,11 @@
 			[ -n "$(CONFIG_USE_APK)" ] && $(STAGING_DIR_HOST)/bin/tar --delete -f ./lib/apk/db/scripts.tar $$(basename $$script); \
 		done; \
 		if [ -z "$(CONFIG_USE_APK)" ]; then \
-			$(if $(IB),,awk -i inplace \
+			awk -i inplace \
 				'/^Status:/ { \
 					if ($$3 == "user") { $$3 = "ok" } \
 					else { sub(/,\<user\>|\<user\>,/, "", $$3) } \
-				}1' $(1)/usr/lib/opkg/status) ; \
+				}1' $(1)/usr/lib/opkg/status ; \
 			$(if $(SOURCE_DATE_EPOCH),sed -i "s/Installed-Time: .*/Installed-Time: $(SOURCE_DATE_EPOCH)/" $(1)/usr/lib/opkg/status ;) \
 		fi; \
 		for script in ./etc/init.d/*; do \
